pkg/mgmt/lvmnode/start.go

```diff
@@ -84,16 +82,7 @@ func Start(controllerMtx *sync.RWMutex, stopCh <-chan struct{}) error {
 	// This lock is used to serialize the AddToScheme call of all controllers.
 	controllerMtx.Lock()
 
-	controller, err := NewNodeControllerBuilder().
-		withKubeClient(kubeClient).
-		withOpenEBSClient(openebsClient).
-		withNodeSynced(nodeInformerFactory).
-		withNodeLister(nodeInformerFactory).
-		withRecorder(kubeClient).
-		withEventHandler(nodeInformerFactory).
-		withPollInterval(60 * time.Second).
-		withOwnerReference(ownerRef).
-		withWorkqueueRateLimiting().Build()
+	nodeCtrller := newNodeController(kubeClient, openebsClientNew, nodeInformerFactory, ownerRef)
```

Comment:
Why are we changing the variable name. The other controllers are named `controller`.
Following consistent semantics will make it easier to read and use tools like `grep`.

Reply:
renamed variables for other controllers appropriately .

Reply:
Using controller for all controller variables.